src/applications/static-pages/dependency-verification/actions/index.js

```diff
@@ -1,23 +1,25 @@
+import { apiRequest } from 'platform/utilities/api';
+// import { isServerError, isClientError } from '../utils';
```

Comment:
These will be used in a future PR when we add the second api call necessary to complete this feature.